src/modules/Elsa.Workflows.Runtime/Contracts/IKeyValueStore.cs

```diff
@@ -0,0 +1,32 @@
+using System.Linq.Expressions;
+using Elsa.Workflows.Runtime.Entities;
+using Elsa.Workflows.Runtime.Models;
+
+namespace Elsa.Workflows.Runtime.Contracts;
+
+/// <summary>
+/// Store that holds key value entities not fit to store in specific stores.
+/// </summary>
+public interface IKeyValueStore
```

Comment:
Unless there's a technical reason this store must live in the Runtime layer, we should move this to its own module (e.g. `Elsa.KeyValues`). Other modules, such as `Elsa.Hosting.Management` are free to take on a dependency to this module.

Reply:
While I agree that the Runtime project is not the right place, I'm not too sure about it having its own project. That could lead to a lot of very small projects in the future.